src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -606,27 +607,6 @@ void CascadiaSettings::_AppendWslProfiles(std::vector<TerminalApp::Profile>& pro
     }
 }
 
-// Function Description:
-// - Get a environment variable string.
-// Arguments:
-// - A string that contains an environment-variable string in the form: %variableName%.
-// Return Value:
-// - a string of the expending environment-variable string.
-std::wstring CascadiaSettings::ExpandEnvironmentVariableString(std::wstring_view source)

```

Comment:
thanks for murding this function

Reply:
Just to be clear, neither Dustin, nor I, nor any other Microsoft employee actually supports murdering any code. All code deserves to live. Even the bad code.